src/org/zaproxy/zap/extension/httpsessions/ExtensionHttpSessions.java

```diff
@@ -182,6 +188,7 @@ public void hook(ExtensionHook extensionHook) {
 			extensionHook.getHookMenu().addPopupMenuItem(getPopupMenuRemoveSession());
 			extensionHook.getHookMenu().addPopupMenuItem(getPopupMenuAddUserFromSession());
 			extensionHook.getHookMenu().addPopupMenuItem(getPopupMenuItemCopySessionToken());
+			extensionHook.getHookMenu().addPopupMenuItem(getPopupMenuItemFindRelatedMessages());
```

Comment:
Consider adding it only if the search extension is enabled.

Reply:
Added enabled check at view check (no point adding panels or menus if it isn't enabled).